### PR TITLE
Tests for the feed(search)/count/schema and search/list/schema were added

### PR DESCRIPTION
**Pull request by mpushnikA1QA**
…dded

### CODE REVIEW

File: src/test/features/сheckCountSchemaEndpoint.feature [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        Examples:
+            | entity                | commonEndpoint |
+            | drug                  | search         |
+            | trial                 | search         |
+            | investigator          | search         |
+            | organization          | search         |
+            | drugCompany           | search         |
+            | drugProgram           | search         |
+            | drugTrends            | search         |
+            | organizationHierarchy | search         |
```
Comment (agromyko) [inline src/test/features/сheckCountSchemaEndpoint.feature:34]: this scenario is the same as previous, can we just remove second scenario and add these examples to the first scenario?

File: src/test/features/сheckListSchemaEndpoint.feature [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@automated @PBPA-1782 @prod
+Feature: List schema endpoint for the search common endpoints for the API entity
+
+    Scenario Outline: Send a request to the ['<entity>'] for the /schema endpoint and make sure that the status code and schema is correct
+        When Send a request to the '<entity>' entity for the search list_schema endpoint
+        Then The API status code should be 200 - OK
+        And Check that schema for the '<entity>' entity for the search _list schema endpoint in the response body is correct
```
Comment (agromyko) [inline src/test/features/сheckListSchemaEndpoint.feature:7]: is there a typo in _list? should it be list_schema as in first step?

File: src/test/features/сheckCountSchemaEndpoint.feature (hunk 1 above)
Comment (mpushnikA1QA) [reply src/test/features/сheckCountSchemaEndpoint.feature]: Update

File: src/test/features/сheckListSchemaEndpoint.feature (hunk 2 above)
Comment (mpushnikA1QA) [reply src/test/features/сheckListSchemaEndpoint.feature]: Fixed

**Review by agromyko — APPROVED** [review]

**Pull request merged**